packages/core/src/TimePicker/enums.js

```diff
@@ -0,0 +1,34 @@
+const TimePickerUnits = {
```

Comment:
all the objects in this enum file could use an Object.freeze() to prevent any undesired modification